src/utils/notifications/checker.ts

```diff
@@ -0,0 +1,139 @@
+import ip from 'ip'
```

Comment:
There is already method for getting the ip address from src/utils/ip.ts, please change it to the methods or remove the utils for uniformity.